backend/backend.go

```diff
@@ -683,3 +685,62 @@ func (backend *Backend) HandleURI(uri string) {
 		backend.log.Warningf("Unknown URI scheme: %s", uri)
 	}
 }
+
+// PocketWidgetSignAddress returns an unused address and makes the user sign a message to prove ownership.
+func (backend *Backend) PocketWidgetSignAddress(code accounts.Code, msg string, format string) (string, string) {
```

Comment:
return (string, string, error) and return all errors and propagate them all the way to the frontend, which should be able to react/show failure modes, if the user aborted, etc.

Reply:
Right, done!